openwisp_radius/tests/test_api/test_api.py

```diff
@@ -703,7 +703,15 @@ def test_api_password_reset(self):
         reset_payload = {'email': 'test@email.com'}
         response = self.client.post(password_reset_url, data=reset_payload)
         self.assertEqual(len(mail.outbox), mail_count + 1)
-
+        email = mail.outbox.pop()
+        self.assertIn(
+            "<p> Please click on the button below to open a page where you can",
+            ' '.join(email.alternatives[0][0].split()),
```

Comment:
why the join and splitting BTW?

Reply:
because the Html contains spaces and `\n`